src/export/packer/next-compiler.ts

```diff
@@ -64,7 +65,7 @@ export class Compiler {
         return zip;
     }
 
-    private xmlifyFile(file: File, prettify?: boolean): IXmlifyedFileMapping {
+    private xmlifyFile(file: File, prettify?: boolean | string): IXmlifyedFileMapping {
```

Comment:
It is now a boolean and a string, what could the string be used for?

Reply:
It seems like it's used like so:

```ts
Packer.toBuffer(doc, '  ').then((buffer) => {
  fs.writeFileSync("document-comments.docx", buffer);
});
```

I would prefer it if it is passed as an option in `File` instead

Reply:
Change the normal string to a enum type to protect the xml output content.